ui/src/components/AdvancedSettingsModal.vue

```diff
@@ -187,6 +187,28 @@
 
           <hr class="advanced-settings-divider" />
 
+          <div>
+            <div class="d-flex justify-content-between align-items-center">
+              <div class="w-75">
+                <label class="mb-0" for="rest">
+                  <p class="font-weight-bold mb-0">Enable REST</p>
+                </label>
+              </div>
+              <div>
+                <toggle-switch
+                  id="rest"
+                  class="align-self-center"
+                  :on="settings.rest"
+                  @toggle="status => (settings.rest = rest)"
```

Comment:
a slight change is needed here for this to work correctly:

```suggestion
                  @toggle="status => (settings.rest = status)"
```

Reply:
thanks, sorry I don't really know what I am doing in frontend things :)

done and squashed in fdb79222a758929278aab08e3879e83c4b70dda5